packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesController.kt

```diff
@@ -271,6 +275,7 @@ class UpdatesController private constructor(
               if (exception == null) {
                 throw AssertionError("Background update with error status must have a nonnull exception object")
               }
+              logger.error("UpdatesController onBackgroundUpdateFinished: Error: ${exception.localizedMessage}", UpdatesErrorCode.None)
```

Comment:
should this (and other) logger.errors have error codes?

Reply:
I think in this case we don't necessarily know what the error was, so we can't assign a code.  Maybe add a new value `UpdatesErrorCode.Unknown` ?

Reply:
Yeah, that seems reasonable.